### PR TITLE
Adding Bookish Invention Project to oss-port along with project map

### PR DESCRIPTION
**Pull request by Shashankjain12**
Added Bookish Invention Project along with the Project Map generated by CodeSee along with labels, maps and tours.

### CODE REVIEW

File: projects/Shashankjain12/bookish-invention.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+*You will see the image that you are preforming the required operations. After that image is being displayed you just need to type any key to follow up further.*
+
+### So Buckle Up Guys Your Notes For Exams Are ready !!
+
+</Overview>
+
+<Contributing>
+  
+For your valuable contribution for the project view respective guidelines under Contributing.md file:-
```
Comment (ramonh) [inline projects/Shashankjain12/bookish-invention.mdx:106]: Ooh! How about linking to the contributing.md file here?

File: projects/Shashankjain12/bookish-invention.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  - Frontend Developers
+  - DevOps
+  - Designers
+  - Maintainers
+  - Code Reviewers
+  - Technical Writers
+  - Mentors
+  - Testers
+featuredMap:
+  url: https://app.codesee.io/maps/50de8840-235b-11ec-a05c-a38053910af1
```
Comment (ramonh) [inline projects/Shashankjain12/bookish-invention.mdx:26]: Is it possible your map isn't set to public?

Could you please [set it to be public](https://docs.codesee.io/en/latest/guide/#sharing-your-map) and share the public URL?

Thanks in advance!

**Review by ramonh — CHANGES_REQUESTED** [review]
Thanks for the submission! 

Left some comments below

File: projects/Shashankjain12/bookish-invention.mdx (hunk 2 above)
Comment (Shashankjain12) [reply projects/Shashankjain12/bookish-invention.mdx:26]: Hi, @ramonh  set the link to public can you please review that again.

File: projects/Shashankjain12/bookish-invention.mdx (hunk 1 above)
Comment (Shashankjain12) [reply projects/Shashankjain12/bookish-invention.mdx]: Added the respective link to it

**Review by ramonh — APPROVED** [review]
Ooooooh, nice Map! Thanks for making it public.

Welcome to Port! 🚢

**Pull request merged**